src/Framework/MockObject/Matcher/ConsecutiveParameters.php

```diff
@@ -37,8 +37,10 @@ class ConsecutiveParameters extends StatelessInvocation
     private $invocations = [];
 
     /**
-     * @throws \PHPUnit\Framework\Exception
```

Comment:
Is the `@throws` annotation not applicable for the constructor anymore?

Reply:
no, it's not applicable.
The constructor doesn't trigger anything that could throw that exception, therefore it was removed.

Reply:
If I understand you correct, removing the annotation is not related to the other changes in this PR. I propose we move it to a separate cleanup PR then.